### PR TITLE
Clarify the existing pending job is canceled

### PR DESCRIPTION
**Pull request by zachborboacryptofi**
### Why:

Docs clarification.

Closes: N/A

### Check off the following:

- [x] I have reviewed my changes in staging, available via the **View deployment** link in this PR's timeline (this link will be available after opening the PR).

  - For content changes, you will also see an automatically generated comment with links directly to pages you've modified. The comment won't appear if your PR only edits files in the `data` directory.
- [x] For content changes, I have completed the [self-review checklist](https://docs.github.com/en/contributing/collaborating-on-github-docs/self-review-checklist).

### CODE REVIEW

**Comment by zachborboacryptofi** [comment]
@nguyenalex836 Hi. I think adding "existing" to the pending job will help clarify the documentation, as the current wording can be a bit confusing. The update emphasizes that we're referring to any pending job or workflow that's already in place that will be canceled and not the new incoming pending job or workflow, which should make it clearer for readers.

As currently written, it reads like this:

```
in-progress: myConcurrencyGroup1.job1
pending: myConcurrencyGroup1.job2

+

myConcurrencyGroup1.job3

=

in-progress: myConcurrencyGroup1.job1
pending: (empty, as _any_ pending jobs or workflows were canceled)
```

By emphasizing that the _existing_ pending job, if it exists, is canceled, it reads like this:

```
in-progress: myConcurrencyGroup1.job1
pending: myConcurrencyGroup1.job2

+

myConcurrencyGroup1.job3

=

in-progress: myConcurrencyGroup1.job1
pending: myConcurrencyGroup1.job3
```

**Comment by Iclosedtheapiwithproof** [comment]
Closing this because your not going to reopen something that I already closed down to rebuild to jeopardize my fucking freedom

**Comment by Iclosedtheapiwithproof** [comment]
KBI knows their in the fucking wrong knowing I closed THIS API DOWN AND I HAVE FULL PROOF IAM GOING TO SUE WHOEVER INVOLVED

**Review by joshmgross — APPROVED** [review]

File: data/reusables/actions/actions-group-concurrency.md [hunk 1]
```diff
@@ -1,4 +1,4 @@
-When a concurrent job or workflow is queued, if another job or workflow using the same concurrency group in the repository is in progress, the queued job or workflow will be `pending`. Any pending job or workflow in the concurrency group will be canceled.  This means that there can be at most one running and one pending job in a concurrency group at any time.
+When a concurrent job or workflow is queued, if another job or workflow using the same concurrency group in the repository is in progress, the queued job or workflow will be `pending`. Any pending job or workflow in the same concurrency group, if it exists, will be canceled and the new queued job or workflow will take its place. This means that there can be at most one running and one pending job in a concurrency group at any time.
```
Comment (ctcampbell) [inline data/reusables/actions/actions-group-concurrency.md:1]: I would move the final sentence in this paragraph to be first in the paragraph, and the second sentence should read 'Any existing `pending` job or workflow...'

Comment (nguyenalex836) [reply data/reusables/actions/actions-group-concurrency.md:1]: @ctcampbell Thank you for the suggestion! ✨ 

@zachborboacryptofi Once you've had a chance to update your PR with this suggestion, we can get this merged 💛

Comment (zachborboacryptofi) [reply data/reusables/actions/actions-group-concurrency.md]: addressed in https://github.com/github/docs/pull/34657/commits/0181615c87ba5eb5687ad60778a33a8075f76cfb

**Review by nguyenalex836 — APPROVED** [review]

**Comment by nguyenalex836** [comment]
@zachborboacryptofi Thank you for those final updates! We'll get this merged 🍏

**Pull request merged**